mono/profiler/proflog.c

```diff
@@ -3096,6 +3103,10 @@ init_suppressed_assemblies (void)
 
 	/* Don't need to free @content as it is referred to by the lines stored in @suppressed_assemblies */
 	content = get_file_content (sa_file);
+	if (content == NULL) {
+		fprintf (stderr, "mono-profiler-log.suppression is greater than 128kb - aborting\n");
+		exit (0);
```

Comment:
Use `g_error` instead here, IIRC `exit` does not work properly on some mobile platforms.